File: app/src/main/java/com/master/kit/testcase/CalendarActivity.java
package com.master.kit.testcase;

import android.os.Bundle;

import com.master.kit.R;
import com.master.kit.widget.calendar.CalendarView;

import tf.oof.com.service.base.BaseSlidingActivity;

public class CalendarActivity extends BaseSlidingActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_calendar);
        CalendarView calendarView = findViewById(R.id.cv_main);
    }
}

File: self-made/src/main/java/com/ooftf/widget/activity/CalendarActivity.kt
package com.ooftf.widget.activity
import android.os.Bundle
import com.alibaba.android.arouter.facade.annotation.Route
import com.ooftf.service.base.BaseSlidingActivity
import com.ooftf.widget.R
import kotlinx.android.synthetic.main.activity_calendar.*

@Route(path = "/widget/calendar")
class CalendarActivity : BaseSlidingActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_calendar)
        calendarView.setOnItemClickListener {
            toast(it.time.toLocaleString())
        }
    }
}
